api/og/postimage/[id].tsx

```diff
@@ -0,0 +1,158 @@
+// @ts-nocheck
+
+import React from 'react';
+
+import type { VercelRequest } from '@vercel/node';
+import { ImageResponse } from '@vercel/og';
+
```

Comment:
Looks like ReactMarkdown sits on top of a https://github.com/remarkjs/remark/tree/main/packages/remark remark unified pipeline.

So I think we may need to use `remark` directly to render Markdown outside React, however I'm not sure what the exact details of the remark pipeline options are, and we'll want to figure this out to ensure that browser and og markdown is identical.

Reply:
not going to render md at this time